### DISCUSSION TITLE
Why does the Button's text become uppercase?

### DISCUSSION THREAD

**Discussion by ccinsz** [beeware/briefcase#1617] [Q&A]
Use the text of Button to change the text, but the Button's text become uppercase. How can it still the same text, not change uppercase?

**Comment by freakboy3742** [comment]
I presume you're referring to the behavior on Android; in which case, that's a property of Android's button widget style - and as a result, there's no way to turn it off.

This has changed relatively recently with the introduction of Material3; however, Toga isn't currently using Material3 widgets. We've only recently started adding  Material3 support as part of adding an OptionContainer widget (see beeware/toga#2350).

**Reply by ccinsz** [reply]
Got it. Thank u for your help.
